Source/WebCore/platform/Scrollbar.cpp

```diff
@@ -286,7 +286,7 @@ void Scrollbar::moveThumb(int pos, bool draggingDocument)
             destinationPosition = std::min(destinationPosition + delta, maximum());
```

Comment:
The `draggingDocument` path are calling with positions, so the minimum and maximum values should consider `scrollOrigin`s when clamping. This should be fixed with a separate patch.